File: text/ocaml/create/text_create.ml
(* ------------------------------------------------------------------- *)
(*
	text_create.ml

					Jul/09/2015

*)
(* ------------------------------------------------------------------- *)
open	Printf
open	Text_manipulate
(* ------------------------------------------------------------------- *)
type unit = {name:string; population:int; date_mod:string}

(* ------------------------------------------------------------------- *)
let data_prepare_proc = fun () ->
	let dict_aa = Hashtbl.create 100
	in let _ = dict_append_proc dict_aa "t2381" "名古屋" 42317 "1995-4-20"
	in let _ = dict_append_proc dict_aa "t2382" "豊橋" 31489 "1995-7-12"
	in let _ = dict_append_proc dict_aa "t2383" "岡崎" 17326 "1995-5-21"
	in let _ = dict_append_proc dict_aa "t2384" "一宮" 82531 "1995-3-24"
	in let _ = dict_append_proc dict_aa "t2385" "蒲郡" 64125 "1995-8-18"
	in let _ = dict_append_proc dict_aa "t2386" "常滑" 72598 "1995-9-8"
	in let _ = dict_append_proc dict_aa "t2387" "大府" 82735 "1995-2-17"
	in let _ = dict_append_proc dict_aa "t2388" "瀬戸" 25476 "1995-1-9"
	in let _ = dict_append_proc dict_aa "t2389" "犬山" 47821 "1995-10-5"
	in
		dict_aa
(* ------------------------------------------------------------------- *)
let _ = print_endline "*** 開始 ***"

let dict_aa = data_prepare_proc ()

let _ = let text_file = Sys.argv.(1)
	in
	printf "%s\n" text_file;
	text_write_proc text_file dict_aa

let _ = print_endline "*** 終了 ***"

(* ------------------------------------------------------------------- *)

File: text/erlang/create/text_create.erl
#! /usr/bin/escript
%% ----------------------------------------------------------------
%%	text_create.erl
%%
%%					Feb/08/2016
%%
%% ----------------------------------------------------------------
-module(text_create).
-import(text_manipulate, [display_proc/1]).
-import(text_manipulate, [text_write_proc/2]).
%% ----------------------------------------------------------------
main(A) ->
	Str_start = unicode:characters_to_binary("*** 開始 ***\n", utf8),
	io:format (Str_start),
	[Arg0|_]=A,
	File_out = Arg0,
	Dict_aa = data_prepare_proc (),
	display_proc (Dict_aa),
%%
	text_write_proc (File_out,Dict_aa),
%%
	Str_end = unicode:characters_to_binary("*** 終了 ***\n", utf8),
	io:format (Str_end).
%%
%% ----------------------------------------------------------------
data_prepare_proc ()->
	[
	{t2381,{{name,"名古屋"},{population,74541},{date_mod,"1996-11-17"}}},
	{t2382,{{name,"豊橋"},{population,67153},{date_mod,"1996-1-14"}}},
	{t2383,{{name,"岡崎"},{population,59286},{date_mod,"1996-5-25"}}},
	{t2384,{{name,"一宮"},{population,38652},{date_mod,"1996-4-15"}}},
	{t2385,{{name,"蒲郡"},{population,72381},{date_mod,"1996-8-30"}}},
	{t2386,{{name,"常滑"},{population,81934},{date_mod,"1996-7-12"}}},
	{t2387,{{name,"大府"},{population,56872},{date_mod,"1996-9-21"}}},
	{t2388,{{name,"瀬戸"},{population,29351},{date_mod,"1996-3-18"}}},
	{t2389,{{name,"犬山"},{population,93456},{date_mod,"1996-7-24"}}}
	].
%% ----------------------------------------------------------------
